### DISCUSSION TITLE
Is it possible to have nested collections?

### DISCUSSION THREAD

**Discussion by lkuoch** [toeverything/AFFiNE#5164] [Q&A] [answered]
I want to be able to nest collections within collections akin to having a organized folder layout.

I'm not able to do this behavior on the UI presently, just wondering if there is a way to enable this or not.

**Comment by EYHN** [comment, marked as answer]
Nested collections should not be possible, but you can create nested backlinks.

![image](https://github.com/toeverything/AFFiNE/assets/13579374/44039734-e83a-4132-9aeb-2b74a6578a5c)

**Reply by lkuoch** [reply]
Would this make sense to support natively?
When you have lots of notes having them in nested collections would definitely improve readability
